### DISCUSSION TITLE
Add settings to disable comment popup when an article is upvoted.

### DISCUSSION THREAD

**Discussion by Ahmdrza** [dailydotdev/daily#382] [Feature Requests]
A setting to disable the comment popup would be great especially in the new tab screen because I just want to read and upvote good articles at that screen.

**Comment by rebelchris** [comment]
I agree this would be great to have as an optional setting.

**Comment by idoshamun** [comment]
Hi @Ahmdrza,

Let me take it back to our team. We do have some plans regarding the card and what should happen before/after reading the article.
I'll keep you posted.
